src/components/ToggleGroup/ToggleGroup.tsx

```diff
@@ -0,0 +1,33 @@
+import * as ToggleGroupPrimitive from '@radix-ui/react-toggle-group'
+import { styled } from '../../stitches.config'
+import { ToggleStyling } from '../Toggle'
+
+const StyledToggleGroup = styled(ToggleGroupPrimitive.Root, {
+  display: 'inline-flex',
+  backgroundColor: 'transparent',
+  borderRadius: '$default',
+})
+
+const StyledItem = styled(ToggleGroupPrimitive.Item, {
+  all: 'unset',
+  ...ToggleStyling,
+
+  '&:first-child': {
+    marginLeft: 0,
+    borderTopLeftRadius: 4,
```

Comment:
border radius should be a design token. If it doesn't work with well with one we have then add another one.